analysers/analyser_osmosis_highway_vs_building.py

```diff
@@ -202,7 +236,7 @@
 """
 
 sql30 = """
-SELECT
+SELECT DISTINCT ON (highway.id, tree.id)
```

Comment:
As tree is a point object, it may be enough to give a single warning (e.g. `DISTINCT ON (tree.id)`), even if two highways are nearby, right? 
If two highways are too nearby, it is very likely that it is the tree (and thus not the highways) is positioned wrong. And solving it for one will then also solve (or modify) the other warning. Just a suggestion, feel free to ignore or postpone to a future PR.

Reply:
Yes.